client/my-sites/site-settings/disconnect-site/troubleshoot.jsx

```diff
@@ -14,35 +14,48 @@ import HelpButton from 'jetpack-connect/help-button';
 import JetpackConnectHappychatButton from 'jetpack-connect/happychat-button';
 import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
 import LoggedOutFormLinks from 'components/logged-out-form/links';
+import { PLAN_JETPACK_FREE } from 'lib/plans/constants';
 import addQueryArgs from 'lib/route/add-query-args';
 import { recordTracksEvent, withAnalytics } from 'state/analytics/actions';
 import { getSiteUrl } from 'state/selectors';
+import { getSitePlanSlug } from 'state/sites/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
 
-const Troubleshoot = ( { siteUrl, trackDebugClick, trackSupportClick, translate } ) => (
+const Troubleshoot = ( { isFreePlan, siteUrl, trackDebugClick, trackSupportClick, translate } ) => (
 	<LoggedOutFormLinks>
 		<LoggedOutFormLinkItem
 			href={ addQueryArgs( { url: siteUrl }, 'https://jetpack.com/support/debug/' ) }
 			onClick={ trackDebugClick }
 		>
 			<Gridicon size={ 18 } icon="offline" /> { translate( 'Diagnose a connection problem' ) }
 		</LoggedOutFormLinkItem>
-		<JetpackConnectHappychatButton
-			label={ translate( 'Get help from our Happiness Engineers' ) }
-			eventName="calypso_jetpack_disconnect_chat_initiated"
-		>
+		{ isFreePlan ? (
 			<HelpButton
 				label={ translate( 'Get help from our Happiness Engineers' ) }
 				onClick={ trackSupportClick }
 			/>
-		</JetpackConnectHappychatButton>
+		) : (
+			<JetpackConnectHappychatButton
+				label={ translate( 'Get help from our Happiness Engineers' ) }
+				eventName="calypso_jetpack_disconnect_chat_initiated"
+			>
+				<HelpButton
+					label={ translate( 'Get help from our Happiness Engineers' ) }
+					onClick={ trackSupportClick }
+				/>
+			</JetpackConnectHappychatButton>
+		) }
 	</LoggedOutFormLinks>
 );
 
 export default connect(
-	state => ( {
-		siteUrl: getSiteUrl( state, getSelectedSiteId( state ) ),
-	} ),
+	state => {
+		const siteId = getSelectedSiteId( state );
+		return {
+			siteUrl: getSiteUrl( state, siteId ),
+			isFreePlan: getSitePlanSlug( state, siteId ) === PLAN_JETPACK_FREE,
```

Comment:
Maybe use `isFreePlan` from `lib/plans` for the `=== PLAN_JETPACK_FREE` logic (since it's already there), or even better - use the `isSiteOnFreePlan` selector after updating it to also account for free Jetpack plans.